admiral/pkg/clusters/registry_test.go

```diff
@@ -187,10 +188,22 @@ func createMockRemoteController(f func(interface{})) (*RemoteController, error)
 	}
 	stop := make(chan struct{})
 	d, e := admiral.NewDeploymentController("", stop, &test.MockDeploymentHandler{}, &config, time.Second*time.Duration(300))
+	if e != nil {
```

Comment:
very nitpicky, this can be ignored, but is it possible to change `e` to `err`?

Reply:
done